ruka/velero/velero.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/env bash
+
+#confirm these variables before installing
```

Comment:
The comments are inconsistent in that some in this file have a space after the `#` and some do.  This repo has consistently been using a space after the `#`.

Reply:
fixed, thanks for noticing it.